### ISSUE TITLE
Command Palette scroll bars not working

### ISSUE THREAD

**Issue by Byteron** [lapce/lapce#704]
When clicking on the scroll bars in the command palette, the palette just closes. Scrolling vertically is possible with the mouse wheel, but scrolling horizontally is not possible that way.

The Palette also closes when clicking on options inside the palette, or the code section that is shown (when using "find all references")

**Comment by MinusGix** [comment]
Good idea, it would be good to have so clicking on previews for things like symbol searching / line searching / multiple reference searching would set that as the current editor.
